webservices/WFLY-19469-Webservice-throttling-feature.adoc

```diff
@@ -0,0 +1,144 @@
+---
+categories:
+- webservices
+stability-level: community
+issue: https://github.com/wildfly/wildfly-proposals/issues/631
+feature-team:
+ developer: ema
+ sme:
+  - ema
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= Add support for webservice throttling feature
+:author:            Jim Ma
+:email:             ema@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+CXF provides throttling feature to limit the request traffic to the websevice endpoint.This new feature to integrate and enable this thorttling feature in WFLY
+webservice subystem.
+
+=== User Stories
+The throttling feature allows server to limit the number of request when the server is busy or there are a lot of faults by some requests. 
+This feature can protect server from being overwhelmed and server resource is used out. 
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-19469[WFLY-19469]
+
+=== Related Issues
+
+- https://issues.redhat.com/browse/EAP7-2223[EAP7-2223]
+
+=== Affected Projects or Components
+
+- https://github.com/jbossws/jbossws-cxf[jbossws-cxf]
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+ 
+== Requirements
+* WildFly webservice subsystem is installed/provisioned and enabled
+* This feature is enabled and configured with jaxws-endpoint-config.xml packaged in user's deployment
+[source,xml]
+ <?xml version="1.0" encoding="UTF-8"?>
+<jaxws-config xmlns="urn:jboss:jbossws-jaxws-config:4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:javaee="http://java.sun.com/xml/ns/javaee"
+  xsi:schemaLocation="urn:jboss:jbossws-jaxws-config:4.0 schema/jbossws-jaxws-config_4_0.xsd">
+  <endpoint-config>
+    <config-name>org.jboss.test.ws.jaxws.cxf.throttling.HelloImpl</config-name>
+    <property>
```

Comment:
Where do these properties get loaded / interpreted?

If this feature is targeting the community stability level my assumption would be that they are disabled at the default stability level - will that be the case?

Reply:
> Where do these properties get loaded / interpreted?

This is loaded and parsed when deploy this deployment. 

> If this feature is targeting the community stability level my assumption would be that they are disabled at the default stability level - will that be the case?

This feature actually is disabled if user doesn't include this deployment descriptor, so I set it to community level.
Should it be default level ?